subsys/testsuite/coverage/coverage.c

```diff
@@ -297,7 +324,7 @@ void gcov_coverage_dump(void)
 struct gcov_info *gcov_get_list_head(void)
 {
 	/* Locking someway before getting this is recommended. */
-	return gcov_info_head;
+    return gcov_info_head;
```

Comment:
[editorconfig](https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig) will guarantee that the right indentation is used